l10n_ar_ux/reports/report_payment_group.xml

```diff
@@ -12,11 +12,20 @@
 
             <t t-set="custom_footer">
                 <div class="row">
-                    <div name="footer_left_column" class="col-8">
+                    <div name="footer_left_column" class="col-4">
                         Observaciones: <span t-field="o.notes"/>
                     </div>
+                    <div name="footer_center_column" class="col-4">
+                        <span>Firma:</span>
+                            <t t-if="request.env['ir.config_parameter'].sudo().get_param('l10n_ar_ux.l10n_ar_report_sig')">
+                                <span class="text-right" t-field="o.company_id.report_signature" t-options='{"widget": "image"}'/>
+                            </t>
+                    </div>
                     <div name="footer_right_column" class="col-4">
-                        <span class="text-center">Firma y Aclaración</span>
```

Comment:
restaurar como estaba antes, pero a este spam le agregamos un "t-if=not o.company_id.report_signature"

antes de cerrar el último div de este reporte, ponemos algo así:
                    <div t-if="o.signature" class="mt32 ml64 mr4" name="signature">
                        <div class="offset-8">
                            <img t-att-src="image_data_uri(company_id.report_signature)" style="max-height: 4cm; max-width: 8cm;"/>
                        </div>
                        <div class="offset-8 text-center">
                            <p t-field="o.company_id.report_signed_by"/>
                        </div>
                    </div>